program/object-detection-tflite/.cm/meta.json

```diff
@@ -41,7 +41,7 @@
   "compiler_env": "CK_CXX",
   "compiler_flags_as_env": "$<<CK_COMPILER_FLAG_CPP11>>$ $<<CK_ENV_LIB_TF_DEF>>$ -Wall -Wno-sign-compare",
   "data_name": "object-detection-tflite",
-  "extra_ld_vars": "$<<CK_ENV_LIB_TF_LINK_OPTIONS>>$ $<<CK_ENV_LIB_TF_LIBS_DIRS>>$ $<<CK_ENV_LIB_TF_LIBS>>$",
+  "extra_ld_vars": "$<<CK_ENV_LIB_TF_LINK_OPTIONS>>$ $<<CK_ENV_LIB_TF_LIBS_DIRS>>$ $<<CK_ENV_LIB_TF_LIBS>>$ -lflatbuffers",
```

Comment:
Ideally, `soft:lib.flatbuffers` should define a new variables. For example, for Boost we have several `CK_ENV_LIB_BOOST_LFLAG_*` variables:
```
anton@diviniti:~/CK_TOOLS/tensorflowmodel-api-master/models$ ck cat env --tags=boost
...
export CK_ENV_LIB_BOOST=/home/anton/CK_TOOLS/lib-boost-1.64.0-gcc-8.2.0-linux-64/install
export CK_ENV_LIB_BOOST_LFLAG_DATE_TIME=-lboost_date_time
export CK_ENV_LIB_BOOST_LFLAG_FILESYSTEM=-lboost_filesystem
export CK_ENV_LIB_BOOST_LFLAG_REGEX=-lboost_regex
export CK_ENV_LIB_BOOST_LFLAG_SYSTEM=-lboost_system
export CK_ENV_LIB_BOOST_LFLAG_THREAD=-lboost_thread
export CK_ENV_LIB_BOOST_SHORT_VER=1.64
export CK_ENV_LIB_BOOST_VER=1.64.0

export CK_ENV_LIB_BOOST_SET=1
```